dependencies.props

```diff
@@ -18,7 +18,7 @@
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <StableVersion Condition="'$(StabilizePackageVersion)' == 'true' and '$(StableVersion)' == ''">$(PackageVersion)</StableVersion>
 
-    <PreReleaseLabel>servicing</PreReleaseLabel>
+    <PreReleaseLabel>preview</PreReleaseLabel>
```

Comment:
should it not be preview1?

Reply:
or preview2 now?

Reply:
Actually, preview2.

Reply:
You plan to do the same in the other repo's?